initialize.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -e
+
+# Check if docker is installed
+if ! [ -x "$(command -v docker)" ]; then
+  echo 'Error: docker is not installed.' >&2
+  # Ask if user wants to install docker
+  read -p "Do you want to install docker? [y/n] " -n 1 -r
+  echo
+  if [[ $REPLY =~ ^[Yy]$ ]]; then
+    # Install docker
+    curl -fsSL https://get.docker.com -o get-docker.sh
+    sh get-docker.sh
+    # Check if docker is installed
+    if ! [ -x "$(command -v docker)" ]; then
+      echo 'Error: Could not install docker.' >&2
+      exit 1
+    fi
+  else
+    exit 1
+  fi
+fi
+
+# Check if docker-compose is installed
+if ! [ -x "$(command -v docker-compose)" ]; then
+  echo 'Error: docker-compose is not installed.' >&2
+  # Ask if user wants to install docker-compose
+  read -p "Do you want to install docker-compose? [y/n] " -n 1 -r
+  echo
+  if [[ $REPLY =~ ^[Yy]$ ]]; then
+    # Install docker-compose
+    sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+    sudo chmod +x /usr/local/bin/docker-compose
+    sudo ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
+    # Check if docker-compose is installed
+    if ! [ -x "$(command -v docker)" ]; then
+      echo 'Error: Could not install docker-compose.' >&2
+      exit 1
+    fi
+  else
+    exit 1
+  fi
+fi
```

Comment:
if it is installed, we should also check the version. See: https://intelowl.readthedocs.io/en/latest/Installation.html#requirements.

In the case the version is old, for docker-compose it would be enough to install the version you installed here over it and that would work

Reply:
@mlodic Overwriting a user's docker-compose (in ```/usr/local/bin/docker-compose```) could lead to unexpected issues with other software as they might be depending on older version

Reply:
ok so we might keep the same approach as the other cases (just telling the user what they should do then). In that way it makes more sense because there is the same behavior for every case.